packages/tradershub/src/components/Timeline/index.tsx

```diff
@@ -13,7 +13,7 @@ type TFormProgressProps = {
  * @param steps - List of steps to be rendered
  * @returns React Component
  */
-export const FormProgress = ({ activeStep, steps = [] }: TFormProgressProps) => {
+export const Timeline = ({ activeStep, steps = [] }: TFormProgressProps) => {
```

Comment:
@shahzaib-deriv May I know why did you change this to timeline? FormProgress makes mare sense to me?

Reply:
I'm thinking in more generic way for components as I've seen it is a timeline in terms of flow 
Ref: https://horizon-ui.com/documentation/docs/data-display/timeline
https://mui.com/material-ui/react-timeline/

We can rename it to form progress as well if required

Reply:
The component will be moved to ui library and in that case it makes more sense to be generic and naming it timeline is better

Reply:
@shahzaib-deriv it has nothing related to time that's why this name is not good generic name, what do you think about `ProgressBar`?

Reply:
Definitely we can rename to ProgressBar